x/mongo/driver/operation.go

```diff
@@ -1925,7 +1844,7 @@ func (op Operation) publishFinishedEvent(ctx context.Context, info finishedInfor
 
 // sessionsSupported returns true of the given server version indicates that it supports sessions.
 func sessionsSupported(wireVersion *description.VersionRange) bool {
-	return wireVersion != nil && wireVersion.Max >= 6
+	return wireVersion != nil
```

Comment:
Does it still make sense to have the logic in a function?

Reply:
I think we should leave it. Here are the pros:

- We check "wireVersion != nil" in many places where determining the session is the purpose, the "sessionSupported" function adds context.
- The definition for session support may mutate with time, give then above point, all we would need to do is extend this function.